### ISSUE TITLE
Add a CC option to notifications

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress#488]
> When I made editorial comments, I'd like the email to also CC to someone else other than the author. Currently, using Who to Notify, the email will send to the checked users (refer image), which will have To: [checked users], which only accurate to Authors, but not to the people I'd like to CC.

![image](https://user-images.githubusercontent.com/3868028/64174007-895a8200-ce26-11e9-87bc-2c9e7759f15e.png)

**Comment by andergmartins** [comment]
Is he suggesting mentioning people in the message, like `@someone` or maybe something like this?

<img width="261" alt="Screen Shot 2019-09-03 at 10 25 07" src="https://user-images.githubusercontent.com/407131/64177083-1d334a80-ce35-11e9-8be5-b61391b217bc.png">

**Comment by sharulhafiz** [comment]
Yes, but for my workflow, I just required a field where I can enter list of email addresses to CC

**Comment by andergmartins** [comment]
Thanks @sharulhafiz for clarifying. Got it 👍
